Lib/test/test_asyncio/test_sendfile.py

```diff
@@ -92,9 +92,13 @@ async def wait_closed(self):
 
 class SendfileBase:
 
-      # 128 KiB plus small unaligned to buffer chunk
-    DATA = b"SendfileBaseData" * (1024 * 8 + 1)
-
+    # 256 KiB plus small unaligned to buffer chunk
+    # Newer versions of Windows seems to have increased its internal 
+    # buffer and tries to send as much of the data as it can as it 
```

Comment:
Nit: I'd try to verify this claim. Perhaps by adding a link to MS API docs, or something similar.

Reply:
Honestly, I doubt if it is reflected in docs. 
Moreover, the socket buffer size seems to depend on the OS build version.